lib/fog/compute/google/models/disk.rb

```diff
@@ -57,15 +57,15 @@ def zone_name
         # auto_delete can only be applied to disks created before instance creation.
         # auto_delete = true will automatically delete disk upon instance termination.
         def get_object(writable = true, boot = false, device_name = nil, auto_delete = false)
-          mode = writable ? "READ_WRITE" : "READ_ONLY"
+          writable ? mode = "READ_WRITE" : mode = "READ_ONLY"
```

Comment:
oh man, that's ugly. Could you expand this out into an `if` statement? I somewhat regret this stylistic choice.